core/src/main/resources/org/elasticsearch/plugins/plugin-install.help

```diff
@@ -20,6 +20,8 @@ DESCRIPTION
 
     The notation of 'username/repository' refers to a github repository.
 
+    The argument can be an valid <url> which points to a download or file location for the plugin to be loaded from.
+
 EXAMPLES
 
     plugin install elasticsearch-analysis-kuromoji
```

Comment:
@dadoonet no I think it is not related, since you are probably more familiar with the implications of changing the artifact names and what else needs to be done, could help with formulating that follow up issue?